### ISSUE TITLE
No Audio Will Play

### ISSUE THREAD

**Issue by Stervan** [nicolasgarnier/drive-music-player#98]
## Expected Behavior
(What did you expect to happen?)
I excpected my music to, once it was put into the player, to play.

## Actual Behavior
(What actually happened?)
The song would load for a couple seconds and then say that the audio file isn't able to be played, although it worked before.

## Steps to Reproduce the Problem

1. Insert any .mp3 file from google drive
2. Let any .mp3 file play automatically or choose a specific .mp3 file to play

## Specifications

- Browser: Google/Microsoft Edge
- OS: Mac/Linux/Windows

**Comment by moonfacedindividual** [comment]
Yes, it might have worked before Stervan but I would guess that you haven't tried since early Feb.  The app has been broken for the past two and a half months with no fix in sight...just read the following posts #82, #83, #84, #84, #85, #86, #87, #88, #89, #90, #91, #92, #93, #94, #95, #96 and #97 .) Sorry for your frustration but we are all in this together.........unfortunately.
